package.json

```
@@ -5,7 +5,7 @@
   "author": "Blaine Bublitz <blaine@grain-lang.org>",
   "license": "Apache-2.0",
   "dependencies": {
-    "ocaml": ">= 4.12.0 < 5.0.0",
+    "ocaml": ">= 4.13.0 < 5.0.0",
```

This should be updated to support 5.0.x releases:
```suggestion
    "ocaml": ">= 4.13.0 < 5.1.0",
```

---

Shouldn't my other pr for supporting ocaml 5 do that?